cmd/kube-proxy/app/server_others.go

```diff
@@ -227,6 +227,7 @@ func newProxyServer(
 				config.IPTables.SyncPeriod.Duration,
 				config.IPTables.MinSyncPeriod.Duration,
 				config.IPTables.MasqueradeAll,
+				config.IPTables.DisableLocalhostNodePorts,
```

Comment:
Switching to just `LocalhostNodePorts` makes it more readable AND shorter

Reply:
Thanks! Done.